### ISSUE TITLE
Make sure tabs in ValueChartViewer are clickable

### ISSUE THREAD

**Issue by MalakronikMausi** [ValueChart/WebValueCharts#191]
Tabs are not clickable in certain situations (when there's a lot of buttons) on Macbook screen because the elements overlap. People should be able to use the system on any laptop. May need to rethink design.

**Comment by TylerJamesMalloy** [comment]
Is this what you meant by 'when there's a lot of buttons' for the tabs?
<img width="411" alt="clustered tabs" src="https://user-images.githubusercontent.com/15621961/31056550-3ea31b18-a688-11e7-806c-8c0f9122c9cc.png">

It seems that whenever the buttons on the right hand side become stacked due to making the page thinner, the 'alternatives' and 'chart details' tab are no longer clickable.

**Comment by MalakronikMausi** [comment]
Yes, but it can also happen in less extreme cases when the buttons are not stacked. Specifically, it can happen when viewing a group chart that you are a member of on a laptop screen. In this case, you also have a lock chart button and a button to toggle between group and individual chart.

**Comment by TylerJamesMalloy** [comment]
So it looks like the valuechart-toolbar that holds the undo, redo, etc buttons is being overlaid on top of the 2 components, the value chart and detail box. And when there isn't enough horizontal space to fit all the buttons, it ends up making the components un-clickable. 

One fast and easy solution is to add style="z-index: 9;" to the toolbar, putting it to the back and making the components clickable even when partially covered by it. I made this edit and I can send it to a test branch for you to check out if you want. 

In the long run we should probably make a solution where the components are better fit into the whole page. I tried a few solutions with bootstrap's grid system but I couldn't manage an easy fix that way. Not totally sure yet how the component system works in Angular, but one solution would be to put the 2 components in a parent component, and make it position absolute to the toolbar so it is pushed down when the toolbar height increases. 

<img width="660" alt="screen shot 2017-10-02 at 4 21 08 pm" src="https://user-images.githubusercontent.com/15621961/31103905-8a433e02-a78e-11e7-806d-9f24c2e8fc65.png">
